src/search-jobs.js

```diff
@@ -2,87 +2,109 @@ const Apify = require('apify');
 const cheerio = require('cheerio');
 
 const { log } = Apify.utils;
-const { BASE_URL, REQUEST_HEADERS } = require('./consts');
 
-const searchJobs = async (query, location, maxResults, proxyUrl) => {
-    const searchEndpoint = '/Job/jobs.htm';
+const { BASE_URL,
+} = require('./consts');
 
-    // global variable for loaded cheerio content to keep jQuery-alike syntax
+const searchJobs = async ({ request }, requestQueue, proxyUrl) => {
+    // GETTING LABEL AND PAGE COUNTER FROM THE REQUEST FOR PAGINATION
+    const { label, searchResults } = request.userData;
+    let { page, itemsToSave, savedItems, maximumResults } = request.userData;
     let $;
-
-    // mapping for items in the jobs search list
+    // FUNCTION TO MAP JOB ELEMS ON THE PAGE
     const mapJobListItem = (i, el) => {
         const employerRating = parseFloat($($($(el).find('div')[0]).find('span')[1]).text());
```

Comment:
this is hard to read, and can be written as `$(el).find('div:eq(0) span:eq(1)').first().text()`